### PR TITLE
⌨️ Correct README, docs typos

### PR DESCRIPTION
**Pull request by swashcap**
* **Problem:** Docs have a few typos!
* **Solution:** Correct misspellings.
* **Testing:** ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The [documentation of Electrode Native] is maintained as [GitBook] pages in the
 
 Please see the [CONTRIBUTING guide](docs/overview/contributing.md)
 
-# Support and Acknowledgement
+# Support and Acknowledgment
```
Comment (swashcap) [inline README.md]: 😜

File: docs/cli/create-runner.md [hunk 2]
```diff
@@ -25,4 +25,4 @@
 * The projects are generated in Android and the iOS directories from the current working directory.  
 * The `platforms` argument is a list of platforms for which to generate the runner. It can either be iOS, Android, or both.  
 * The `mainMiniAppName` argument is the name of the main MiniApp. This is the MiniApp that should get launched when the Runner is started.  
-* There isn't an `ern` command that launches the generated Runners.  To manually launch the generated Runner projects, use Android Studio or XCode to compile and launch the Runners—or use appropriate commands to open the generated Runner projects.
```
Comment (swashcap) [inline docs/cli/create-runner.md]: Apple refers to it as [“Xcode”](https://developer.apple.com/xcode/), so good to follow their lead.

File: docs/getting-started/getting-started.md [hunk 3]
```diff
@@ -454,7 +454,7 @@ import { NavigationApi } from 'react-native-ernnavigation-api'
         style={styles.container}
         dataSource={this.state.dataSource}
         renderRow={(movie) =>
-        <TouchableHighlight onPress={() => this._onPressRow(movie)} underlayColor="grey">
+        <TouchableHighlight onPress={() => this._onPressRow(movie)} underlayColor="gray">
```
Comment (swashcap) [inline docs/getting-started/getting-started.md]: en-GB → en-US

**Comment by belemaire** [comment]
🎉 

wow. Thanks a lot for this 👍

File: docs/overview/what-is-ern.md [hunk 4]
```diff
@@ -6,21 +6,21 @@ Electrode Native is built on top of [React Native](https://facebook.github.io/re
...
 
-To view the platform code percentage of Javascript code versus Native code, a good approximation would be 70% JavaScript and 30% native code.  
+To view the platform code percentage of JavaScript code versus Native code, a good approximation would be 70% JavaScript and 30% native code.
 
-Electrode Native is composed of several modules that are accessed using the Electrode Native CLI client. The CLI client is the only tool that developers need when using Electrode Native. Electrode Native runs on Node v4.5 or later. Electrode Native runs on Mac OSX only. Windows support is scheduled to follow. Electrode Native also runs on Linux; however only minimal testing has verified this support.
+Electrode Native is composed of several modules that are accessed using the Electrode Native CLI client. The CLI client is the only tool that developers need when using Electrode Native. Electrode Native runs on Node v4.5 or later. Electrode Native runs on Mac OS X only. Windows support is scheduled to follow. Electrode Native also runs on Linux; however only minimal testing has verified this support.
```
Comment (swashcap) [inline docs/overview/what-is-ern.md]: Could also update to “macOS” now that it’s called that.

File: docs/platform-parts/cauldron.md [hunk 5]
```diff
@@ -102,13 +102,13 @@ The configuration also shows the different objects stored in the cauldron--level
 
 * For each mobile application, the second level is the platforms array. Electrode Native supports two platforms: Android and iOS. For each platform, there can be multiple versions of a mobile application. At this level (mobile application + platform + version), most of the Cauldron data resides.  
 
-* At the Android platform level of the `MyWeatherApp` application, a configuration object contains data that applies to every version of the `MyWeatherApp` for the Android platform. It also contains CodePush configurations. For information about CodePush, see the [CodePush documentation](url) for more details. For information about the containuer generator configuration, see the [Container documentation](url).
+* At the Android platform level of the `MyWeatherApp` application, a configuration object contains data that applies to every version of the `MyWeatherApp` for the Android platform. It also contains CodePush configurations. For information about CodePush, see the [CodePush documentation](url) for more details. For information about the container generator configuration, see the [Container documentation](url).
```
Comment (swashcap) [inline docs/platform-parts/cauldron.md]: :scream: Holy GitHub diffs. Might be better to diff this file via CLI.

**Pull request merged**